app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,366 @@
+
+    <string name="label_generic_xmrto_error">ERREUR XMR.TO</string>
+    <string name="text_generic_xmrto_error">Code : %1$d</string>
+
+    <string name="text_retry">Touchez pour réessayer</string>
+    <string name="text_noretry_monero">Maintenant on est coincé ici !</string>
+    <string name="text_noretry">Oh-oh, XMR.TO n\'a pas laire disponible pour le moment !</string>
+
+    <string name="text_send_btc_amount">%1$s BTC = %2$s XMR</string>
+    <string name="text_send_btc_rate">(Taux : %1$s BTC/XMR)</string>
+
+    <string name="label_send_settings_advanced">Avancé :</string>
+
+    <string name="label_send_btc_details">Cliquez pour plus d\'infos</string>
+    <string name="label_send_btc_xmrto_info">Visitez xmr.to pour l\'assistance &amp; le suivi</string>
+    <string name="label_send_txinfo">Voir les détails TX pour plus d\'infos</string>
+    <string name="label_send_btc_xmrto_key_lb">Clef Secrète\nXMR.TO</string>
+    <string name="label_send_btc_xmrto_key">Clef Secrète XMR.TO</string>
+    <string name="label_send_btc_address">Adresse BTC Destination</string>
+    <string name="label_send_btc_amount">Montant</string>
+    <string name="label_send_btc_create_order">Création de l\'ordre XMR.TO</string>
+
+    <string name="label_send_btc_retry">Touchez pour réessayer</string>
+
+    <string name="label_send_txid">ID de Transaction</string>
+    <string name="label_send_address">Adresse Destination</string>
+    <string name="label_send_payment_id">ID de Paiement</string>
+    <string name="label_send_notes">Notes</string>
+
+    <string name="backup_progress">Sauvegarde en cours</string>
+    <string name="archive_progress">Archivage en cours</string>
+    <string name="rename_progress">Renommage en cours</string>
+    <string name="open_progress">Vérification de la connexion au démon</string>
+    <string name="changepw_progress">Modification du mot de passe en cours</string>
+
+    <string name="service_progress">Récapitulation &#8230;\nCela peut prendre un moment !</string>
+
+    <string name="backup_success">Sauvegarde réussie</string>
+    <string name="backup_failed">Sauvegarde Échouée !</string>
+    <string name="archive_success">Archivage réussi</string>
+    <string name="archive_failed">Archivage Échouée !</string>
+    <string name="delete_failed">Suppression Échouée !</string>
+    <string name="rename_failed">Renommage Échoué !</string>
+    <string name="changepw_failed">Modification du mot de passe Échouée !</string>
+    <string name="changepw_success">Modification du mot de passe réussie</string>
+
+    <string name="label_daemon">Nœud</string>
+    <string name="prompt_daemon">([&lt;utilisateur&gt;:&lt;mdp&gt;@]&lt;démon&gt;[:&lt;port&gt;])</string>
+    <string name="prompt_mainnet">Sélection Réseau</string>
+    <string name="connect_stagenet">StageNet</string>
+    <string name="connect_testnet">TestNet</string>
+    <string name="connect_mainnet">MainNet</string>
```

Comment:
shouldn't it just be "Stagenet", "Testnet", and "Mainnet"? That's the way it is done in the official CLI/GUI

Reply:
damned!

Reply:
I can lowercase the net part in every language if you're ok.

Reply:
IMO they should be the same in all languages. I'll do the change in values/strings.xml, add the untranslatable to it, and remove the three strings from other strings.xml. (after a rebase for norvegian)